ndb/src/google/cloud/ndb/key.py

```diff
@@ -1241,3 +1247,11 @@ def _to_legacy_path(dict_path):
         elements.append(element)
 
     return _app_engine_key_pb2.Path(element=elements)
+
+
+class GetByKeyFuture(_future.Future):
+    """Future for looking up entities by Key."""
+
+    def _compute_result(self, result):
```

Comment:
What type is `result`?

Reply:
Detail to be fleshed out later. Whatever is returned by stub.Lookup.future().get_result(), in this case a datastore_pb2.LookupResponse